Resources/Prototypes/Catalog/VendingMachines/Inventories/winterdrobe.yml

```diff
@@ -13,9 +13,7 @@
     ClothingOuterCoatBomber: 3
     ClothingHeadHatSantahat: 2
     ClothingHeadHatXmasCrown: 2
-    ClothingNeckScarfStripedCentcom: 1
 
   emaggedInventory:
     ClothingNeckScarfStripedSyndieGreen: 3
     ClothingNeckScarfStripedSyndieRed: 3
-    ClothingNeckScarfStripedCentcom: 3
```

Comment:
Is it not intentional for the emag to unlock Centcom scarfs?

Reply:
wasn't sure, reverted